Description: Escape manpage hyphens
 Escape hyphens in manpage to avoid hyphen-used-as-minus-sign lintian warning.
Author: Doug Torrance <dtorrance@monmouthcollege.edu>
Last-Update: 2014-10-05

--- a/wmbattery.1x
+++ b/wmbattery.1x
@@ -114,7 +114,7 @@
 .SH EXAMPLE
 Start at 10% battery to execute 'echo' including status information:
 .RS
-wmbattery -c 10 -x "echo Status: %percent%% - %minutes% minutes, %seconds% seconds left"
+wmbattery \-c 10 \-x "echo Status: %percent%% \- %minutes% minutes, %seconds% seconds left"
 .RE
 .SH AUTHOR
 Joey Hess <joey@kitenet.net>
